ksql-engine/src/test/java/io/confluent/ksql/function/UdfClassLoaderTest.java

```diff
@@ -64,4 +64,4 @@ public void shouldLoadNonConfluentClassesFromChildFirst() throws ClassNotFoundEx
         not(org.slf4j.impl.Log4jLoggerAdapter.class));
   }
 
-}
\ No newline at end of file
+}
```

Comment:
I think its just whitespace. I was fiddling around with this test because it relies on the slf4j implementation and I was trying to clean that up, but decided to punt that to another PR. I guess this slipped in.